AppCenterDemoApp/Assets/Puppet/PuppetCrashes.cs

```diff
@@ -222,4 +237,10 @@ private IEnumerator LastCrashReportCoroutine()
             LastSessionCrashReport.text = "App has not crashed during the last session";
         }
     }
+
+    [DllImport("PuppetBreakpad")]
+    private static extern void nativeCrashNullPointer();
+
+    [DllImport("PuppetBreakpad")]
+    private static extern void setupNativeCrashesListener(string path);
```

Comment:
Let's move it to separate helper class

Reply:
The definition? I don't fully agree on this one

Reply:
okay, not big deal